vizro-core/docs/pages/user-guides/custom-components.md

```diff
@@ -266,6 +266,224 @@ vm.Page.add_type("components", Jumbotron)
     [CustomComponent2]: ../../assets/user_guides/custom_components/customcomponent_2.png
 
 
+## Custom component with custom action
+
+Custom components can be used as `inputs` to, `outputs` of, or as a `trigger` of custom actions. In the examples below we will explore both options.
+
+### Custom component as input/output of custom action
```

Comment:
```suggestion
### Custom components as inputs/outputs of custom actions
```

Reply:
This could also be a "how to" header but I suspect it'll make it long in the table of contents, so I'm happy to leave it like this (but use the same form as in the paragraph above, so changed to plurals here).